src/cloud/linode/deploy_diarkis_on_linode.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+  echo "Usage: TF_VAR_token=<token> $0 [workspace] [ROOT_DIR]"
+  echo "  workspace (optional): The Terraform workspace to use. (default: dev-diarkis-asia)"
+  echo "  ROOT_DIR (optional): The root directory of the project."
+  echo "  TF_VAR_token: The Linode Personal Access Token."
+  echo
+}
+
+if [ "$1" == "-h" ] || [ "$1" == "--help" ]; then
+  usage
+  exit 0
+fi
+
+# Set workspace_name as the first argument or default to dev-diarkis-asia
+workspace_name=${1:-dev-diarkis-asia}
+
+# Set ROOT_DIR as the first argument or default to the root directory of the project
+ROOT_DIR=${2:-$(cd $(dirname $0)../../; pwd)}
```

Comment:
```
$ pwd
src/cloud/linode
$ ./deploy_diarkis_on_linode.sh zzz
./deploy_diarkis_on_linode.sh: line 22: cd: .../../: No such file or directory
Error: Token is required. Provide it as the first argument or set TF_VAR_token.
```

Reply:
Why not using
```
SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
ROOT_DIR=${2:-$(cd ${SCRIPT_DIR}/../../; pwd)}
```

Reply:
oh right, what i wanted to do is like this
```
ROOT_DIR=$(cd $(dirname $0); cd ../../; pwd)
```

and actually realised there's no meaning to take the path from args as this script always need to consider the ROOT_DIR is the project root